WordPress/Classes/ViewRelated/Reader/ReaderFollowedSitesViewController.swift

```diff
@@ -202,25 +203,25 @@ class ReaderFollowedSitesViewController: UIViewController, UIViewControllerResto
             return
         }
 
-        let generator = UINotificationFeedbackGenerator()
-        generator.prepare()
-
         let service = ReaderSiteService(managedObjectContext: managedObjectContext())
         service.followSite(by: url, success: { [weak self] in
-            let success = NSLocalizedString("Followed", comment: "User followed a site.")
-            SVProgressHUD.showDismissibleSuccess(withStatus: success)
-            generator.notificationOccurred(.success)
+            SVProgressHUD.dismiss()
```

Comment:
Why is this line needed?

Reply:
You're right, this and the `import` can be removed 👍